src/test/java/org/web3j/console/project/unit/gen/UnitClassGeneratorTest.java

```diff
@@ -27,6 +27,24 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class UnitClassGeneratorTest extends Setup {
+    private static String classAsString;
+    private static File classAsFile =
+            new File(
+                    temp
+                            + separator
```

Comment:
Might improve readability to use a more concise syntax:

`String.join(File.separator, "a", "b", "c");`